src/pages/[platform]/build-a-backend/data/set-up-data/index.mdx

```diff
@@ -623,6 +679,56 @@ export default function TodoList() {
 }
 ```
 
+</InlineFilter>
+
+<InlineFilter filters={["vue"]}>
+
+```html title="src/TodoList.vue"
+
+<script setup lang="ts">
+import { onMounted, ref } from 'vue';
+import type { Schema } from '../../amplify/data/resource'
+import { generateClient } from 'aws-amplify/data'
+
+const client = generateClient<Schema>()
+
+// create a reactive reference to the array of todos
+const todos = ref<Array<Schema['Todo']["type"]>>([]);
```

Comment:
quotes should be consistent -- single or double

Reply:
👍